_projects/2019/juliaastro/astroimages.md

```diff
@@ -0,0 +1,69 @@
+---
+name: Visualization of astronomical images in Julia
+desc: Expand the capabilities of the AstroImages.jl package
+requirements:
+ - Astronomy
```

Comment:
is astronomy required though for this project?

Reply:
Well, not astronomy as a whole but at least familiarity with FITS files would be appreciated.

Reply:
One doesn't ensure the other (I suppose there are more astronomy students unfamiliar with FITS than the ones familiar enough for the project).
 But it's @dpshelio's call, I'm happy to have this merged as is or with removing the astronomy here.

Reply:
Changed to "familiarity with FITS file", which is the real requirement